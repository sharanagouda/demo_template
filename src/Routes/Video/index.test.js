import Video from "./index";

jest.mock("@discovery-dni/np2", () => ({
  addMedia: {}
}));

const location = {
  pathname: "/video/goldrausch-in-alaska/1000-unzen"
};

const data = {
  page: {
    primaryContent: {
      type: "video"
    }
  }
};

test("renders div", () => {
  const component = new Video({ location, data }).render();

  expect(component).toMatchSnapshot();
});
