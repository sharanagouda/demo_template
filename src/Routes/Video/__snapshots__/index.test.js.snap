// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders div 1`] = `
Object {
  "addClass": [Function],
  "appendTo": [Function],
  "element": <div />,
  "isMounted": false,
  "prependTo": [Function],
  "remove": [Function],
  "removeClass": [Function],
}
`;
